charts/grafana/templates/dashboards-json-configmap.yaml

```
@@ -9,6 +9,9 @@ metadata:
   labels:
     {{- include "grafana.labels" $ | nindent 4 }}
     dashboard-provider: {{ $provider }}
+    {{- if $.Values.sidecar.dashboards.enabled }}
+    {{ $.Values.sidecar.dashboards.label }}: {{ $.Values.sidecar.dashboards.labelValue }}
```

@zanhsieh We should set the `labelValue` by default to `1` so that it does not try to create a label with an empty string value (or alternatively use helm `default` function). This broke the chart for me.

---

And parse the value to `quote` to ensure it is a string

```suggestion
    {{ $.Values.sidecar.dashboards.label }}: {{ $.Values.sidecar.dashboards.labelValue | default "1" |  quote }}
```

---

Now I need to enter

```yaml
        sidecar:
          dashboards:
            enabled: true
            label: grafana_dashboard
            labelValue: "'1'"
```

---

@sjoukedv applied your suggestion in https://github.com/grafana/helm-charts/pull/2693